pkg/rack/console.go

```diff
@@ -256,7 +256,12 @@ func (c Console) UpdateVersion(version string) error {
 }
 
 func isSkippingMinor(currentVersion, version string) error {
-	cmv, err := strconv.Atoi(strings.Split(currentVersion, ".")[1])
+	splittedVersion := strings.Split(currentVersion, ".")
+	if len(splittedVersion) == 0 {
+		return nil
+	}
+
+	cmv, err := strconv.Atoi(splittedVersion[0])
```

Comment:
shouldn't it be `splittedVersion[1]` instead of `splittedVersion[0]`

Reply:
also i think we need slice len check in here too `mv, err := strconv.Atoi(strings.Split(version, ".")[1])`

Reply:
thanks for the feedback, I forgot about it 😄